docs/spec/api.md

```diff
@@ -1,7 +1,5 @@
 # Docker Registry HTTP API V2
 
-## Introduction
```

Comment:
Specs != generic docs, either way.
@moxiegirl can we let specs out of scope for now? (sent a mail about that)
Thanks.